nautobot/core/tests/test_api.py

```diff
@@ -631,6 +631,7 @@ def setUp(self):
         self.vlan_group2 = ipam_models.VLANGroup.objects.create(name="Test VLANGroup 2", location=self.location2)
         self.vlan_group3 = ipam_models.VLANGroup.objects.create(name="Test VLANGroup 3", location=self.location3)
 
+    @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
```

Comment:
I'd prefer (even though it's more work) to only add EXEMPT_VIEW_PERMISSIONS to the generic tests in `testing/api.py` (since those are the ones with downstream impact to Apps). All of the core-exclusive tests like this one should instead be fixed with correct object permissions instead.